src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -52,10 +52,10 @@
 #include "mongoBackend/connectionOperations.h"
 #include "mongoBackend/TriggeredSubscription.h"
 #include "cache/subCache.h"
-
+#include "rest/StringFilter.h"
 #include "ngsi/Scope.h"
 #include "rest/uriParamNames.h"
-
+#include "rest/StringFilter.h"
```

Comment:
Twice the same include? I guess this is a typo...

(Offtopic: it is a pity C/C++ doesn't have any mechanism to detect this redundancy... :( )